### PR TITLE
Migrate fraud detection notebook 

### PR DESCRIPTION
**Pull request by xuex2017**
run fraudDetection notebook on zoo

### CODE REVIEW

**Comment by shane-huang** [comment]
Please ask dongjie and configure integration test correctly for this. If jenkins test passed, add a comment to this pr so that we can merge it.

**Comment by xuex2017** [comment]
jenkins passed http://172.168.2.101:8080/job/ZOO-PR-Validation/272/  @shane-huang

File: notebook/scala/fraudDetection/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0/snapshots/dev1/toree-pip/toree-0.2.0.dev1.tar.gz
+```
+
+2. Build zoo jar file under Spark 2.x
+
+3. To support the training for imbalanced data set in fraud detection, some Transformers and algorithms are developed in 
+https://github.com/intel-analytics/analytics-zoo/tree/legacy/pipeline/fraudDetection.
+
+4. build frauddetection jar file with maven under the directory or use fraud-1.0.1-SNAPSHOT.jar
+
```
Comment (hhbyyh) [inline notebook/scala/fraudDetection/README.md]: add: One pre-built version is provided in this directory.

File: notebook/scala/fraudDetection/Fraud Detction.ipynb [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "By using bagging model, we can increase the AUPRC from 0.71 to 0.93.\n",
+    "\n",
+    "The code used in this notebook can be found from https://github.com/intel-analytics/analytics-zoo/tree/master/pipeline/fraudDetection.\n",
```
Comment (hhbyyh) [inline notebook/scala/fraudDetection/Fraud Detction.ipynb]: update link

**Comment by xuex2017** [comment]
@hhbyyh updated README.md and notebook

**Pull request closed without merging**